spec/models/purchase_spec.rb

```diff
@@ -1265,6 +1265,12 @@ def verify_balance(user, expected_balance)
       purchase = create(:purchase, price_cents: 120, total_transaction_cents: 120, tax_cents: 20, zip_tax_rate:, was_tax_excluded_from_price: true)
       expect(purchase.tax_label).to eq("Sales tax")
     end
+
+    it "exported tax label properly" do
+      zip_tax_rate = create(:zip_tax_rate, country: "AU", combined_rate: 0.1)
+      purchase = create(:purchase, price_cents: 100, total_transaction_cents: 110, gumroad_tax_cents: 10, zip_tax_rate:)
+      expect(purchase.tax_label(include_tax_rate: false)).to eq("GST")
+    end
```

Comment:
@ershad does this look reasonable?

<img width="455" alt="Screenshot 2025-06-22 at 3 45 50 PM" src="https://github.com/user-attachments/assets/efcd2fcd-9159-4934-a648-2a80e905cffa" />

Reply:
@chaitanyya let's add for `VAT` and `Sales tax` too? Maybe we can organize tests like this to cover all cases:

```
  describe "tax_label"
    describe "GST"
      context "when include_tax_rate is true"
      context "when include_tax_rate is false"

    describe "VAT"
      context "when include_tax_rate is true"
      context "when include_tax_rate is false"

    describe "Sales tax"
      context "when include_tax_rate is true"
        context "when was_tax_excluded_from_price is true"
        context "when was_tax_excluded_from_price is false"
      context "when include_tax_rate is false"
```

Reply:
Extended test coverage:

<img width="1020" alt="Screenshot 2025-06-22 at 4 20 38 PM" src="https://github.com/user-attachments/assets/e43490b2-73ba-4326-a499-f29a0f01f1cf" />